### ISSUE TITLE
Better way for long Drop down at the navbar

### ISSUE THREAD

**Issue by juanetec** [Servoy/svyNavigation#8]
Hi,
I am looking for a better solution for the drop down at the navbar. In smart we have the popupmenu that we can nested with  "popupmenu.addMenu.". I know that the bootstrap Drop down cannot do it.
I saw that option [http://bootstrapessentials.com/fulldocs/components/navbar/navbar-submenu/](url) but I don't know how to implementent it in the {CustomType<bootstrapextracomponents-navbar.subMenuItem>}.

I can show a screen shot of may case is better to understand what I am trying to say:
![Image 7](https://user-images.githubusercontent.com/26114868/77903333-93d4e100-727a-11ea-89e1-afb73fe2cce2.png)

Thanks
Un Saludo

Juan Ramírez

**Comment by seanthomasdevlin** [comment]
Hi Juan,

Yes it is possible to do nested menus on the [NavBar component](https://github.com/Servoy/bootstrapextracomponents/wiki/Navbar) that is included in the navigation module. Use the **subMenuItems** property.

In the context of the navigation module, you can override the [loadNavbarItems](https://github.com/Servoy/svyNavigation/wiki#init-the-navbar) method. There are instructions on the svyNavigation wiki.

**Comment by juanetec** [comment]
Hi Sean,
sorry about my another issue maybe I did not explain clearly , 
what you have explained is that what I do. 
But what I pretend is to nested more than one level.
In this example I have only one level
Can I have the level 6.2.1? on the next code example.
`var menuItems = [];
menuItems.push({itemId: '6', text: 'Menu', position: 'RIGHT', tooltip: 'A submenu', subMenuItems: [{itemId: '6.1', text: 'Action'}, {itemId: '6.2', text: 'More action'}, {isDivider: true}, {itemId: '6.3', text: 'Last action'}]});
elements.navbar.brandText = 'Accounting';
elements.navbar.setMenuItems(menuItems);`


I found a boostrap navbar submenu example at that link and they solve it with StyleClass
![Image 2](https://user-images.githubusercontent.com/26114868/78000785-ba535480-732c-11ea-8f43-c660d5328133.png)

https://bootstrapessentials.com/fulldocs/components/navbar/navbar-submenu/


Thanks
 Un Saludo

Juan Ramírez
